packages/wrangler/src/autoconfig/run.ts

```diff
@@ -47,40 +51,84 @@ export async function runAutoConfig(
 		});
 	}
 
+	autoConfigDetails = updatedAutoConfigDetails;
+
 	const deploy = await confirm(
 		"Do you want to proceed with the deployment using these settings?"
 	);
```

Comment:
I don't think we need this question anymore. The wording is good, but we don't need two confirmations dialogs before deploying (this and the one in `buildAndConfirmOperationsSummary()`)

Reply:
I think it would be good to print all the planned changes at this point, like we do below. It would be good if there were a way to log what `configure()` commands would be run too.